packages/react-testing/README.md

```diff
@@ -438,6 +438,18 @@ expect(myComponent).toContainReactComponent('div', {
 });
 ```
 
+### `.toContainReactComponentTimes(type: string | React.ComponentType, times: number, props?: object)`
```

Comment:
3 positional params feels a bit :/ but I can't really think of a better way to do this API

Reply:
Same here. I think we should still expose prop matching as third param (optional).